pkgs/by-name/te/ted/package.nix

```
@@ -110,9 +114,8 @@ stdenv.mkDerivation rec {
       Acrobat PDF converter.
     '';
     homepage = "https://nllgg.nl/Ted/";
```

```suggestion
    homepage = "https://ftp.nluug.nl/pub/editors/ted/";
```
Original homepage is a 404.